tests/actions/IOUTest.js

```diff
@@ -2021,6 +2022,7 @@ describe('actions/IOU', () => {
 
             // Given fetch operations are resumed
             fetch.resume();
+            await waitForBatchedUpdates();
```

Comment:
@fedirjh 
We need to wait for the onyx updates to complete so that tests succeed. Earlier, it didn't matter as we were optimistically deleting the IOU report action. Now, we delete only on success.